<launch>
  <node
    pkg="autocycle"
    type="start_nav"
    name="navigation_core"
    output="screen"
    required="true"
  />
  <node
    pkg="autocycle"
    type="run_lidar"
    name="run_lidar"
    output="screen"
    required="true"
  />
  <node
    pkg="autocycle"
    type="get_deltas"
    name="get_deltas"
    output="screen"
    respawn="true"
  />
  <node
    pkg="autocycle"
    type="calculate_deltas"
    name="calculate_deltas"
    output="screen"
    respawn="true"
  />
  <node
    pkg="autocycle"
    type="read_serial"
    name="read_serial"
    output="screen"
    respawn="true"
  />
  <node
    pkg="autocycle"
    type="test_fake_send_action"
    name="test_fake_send_action"
    output="screen"
    required="true"
  />
  <node
    pkg="autocycle"
    type="frame_ready"
    name="frame_ready"
    respawn="true"
  />
</launch>
